testnet.json

```diff
@@ -11,7 +11,7 @@
         "flow_token": "7e60df042a9c0868",
         "fungible_token": "9a0766d93b6608b7"
     },
-    "data_dir": "/data",
+    "data_dir": "./data",
```

Comment:
why do we need to change this?

Reply:
I ran into issues on OSX without this change